LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-active-document.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<link rel="author" href="mailto:masonf@chromium.org">
+<link rel=help href="https://github.com/whatwg/html/pull/10705">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  test(() => {
+    const doc = document.implementation.createHTMLDocument();
+    const popover = doc.createElement('div');
+    popover.setAttribute('popover','');
+    doc.body.appendChild(popover);
+    assert_throws_dom('InvalidStateError',() => popover.showPopover());
+    assert_false(popover.matches('[open]'));
```

Comment:
The WPT changes, had this as `:open` — which is not a valid CSSSelector, so the tests failed. So changed it to this:

```diff
-    assert_false(popover.matches(':open'));
+    assert_false(popover.matches('[open]'));
```

Is it worth upstreaming this change as well? If so, I'll action that.

Reply:
I think you want `:popover-open`, `:open` isn't defined on popovers. Popovers also don't set the `[open]` attribute.